diff --git a/CMakeLists.txt b/CMakeLists.txt
index 22bcd44..0c71ed0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -481,6 +481,8 @@ add_subdirectory(interface/basisu_c_binding)
 # this CMakeLists is included in another project which is unlikely
 # except for building the ktx library.
 if((KTX_FEATURE_TOOLS OR KTX_FEATURE_TESTS) AND NOT TARGET fmt::fmt)
+    find_package(fmt CONFIG REQUIRED)
+elseif(0)
     # Always build fmt static
     set(BUILD_SHARED_LIBS OFF)
     set(FMT_INSTALL OFF)
@@ -489,7 +491,7 @@ if((KTX_FEATURE_TOOLS OR KTX_FEATURE_TESTS) AND NOT TARGET fmt::fmt)
     set(BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS_RESET})
 endif()
 if(KTX_FEATURE_TOOLS AND NOT TARGET cxxopts::cxxopts)
-    add_subdirectory(external/cxxopts)
+    find_package(cxxopts CONFIG REQUIRED)
 endif()
 
 # Tools
