diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index e18b837..2f0c6ec 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -218,7 +218,7 @@ macro(common_libktx_settings target enable_write library_type)
         XCODE_ATTRIBUTE_ENABLE_HARDENED_RUNTIME "YES"
     )
 
-    if(APPLE_LOCKED_OS)
+    if(0)
         set_target_properties(${target} PROPERTIES
             FRAMEWORK TRUE
         )
